views/js/admin/src/components/order-list-column/LabelCard.vue

```diff
@@ -9,7 +9,7 @@
         v-text="label.barcode" />
       <a
         class="btn btn-link"
-        @click="() => print(label)">
+        @click="() => print(label) & refresh(label)">
```

Comment:
Maak hier maar gewoon een method van waarin je dit netjes in 2 regels doet ipv deze creepy syntax